timify-api/server.js

```diff
@@ -0,0 +1,61 @@
+import express from 'express';
+import session from 'express-session';
+import cors from 'cors';
+import morgan from 'morgan';
+import { sequelize } from './database.js';
+import { User} from './models/index.js';
+import userRoutes from './route/users.js';
+import SequelizeStoreInit from 'connect-session-sequelize';
+
+const app = express();
+
+app.use(cors({
+  origin:'http://localhost:5173 ',
+  credentials: true
+}));
+app.use(express.json());
+app.use(morgan())
+
+const SequelizeStore = SequelizeStoreInit(session.Store);
+const sessionStore = new SequelizeStore({
+  db: sequelize
+});
+
+// Session middleware
+app.use(
+  session({
+    secret: 'your-secret-key',
```

Comment:
It's good practice never to hard code something (meaning we don't want to directly put values into code) so if you could define the key in a variable and use that here, would be great!